qiling/os/posix/syscall/epoll.py

```diff
@@ -226,7 +224,7 @@ def ql_syscall_epoll_wait(ql: Qiling, epfd: int, epoll_events: int, maxevents: i
 
         # FIXME: the data packed after events should be the one passed on epoll_ctl
         # for that specific fd. currently this does not align with the spec
-        data = ql.pack32(events) + ql.pack64(fd)
```

Comment:
According to [Linux kernel code](https://elixir.bootlin.com/linux/v6.14.4/source/include/uapi/linux/eventpoll.h#L83) `events` is a form of an `int`, and `data` is of type `__u64`, so data is always 64-bit in size. The code uses a macro that conditionally expands to alignment based on whether this is a 32 or 64 bit arch. In any case the size of the structure is 12 bytes